src/main/java/com/gravity9/mongocdc/MongoChangeStreamWorker.java

```diff
@@ -129,10 +129,15 @@ public void run() {
 
                 // Read resumeToken even with no new results to make sure the token does not expire
                 BsonDocument resumeTokenDoc = cursor.getResumeToken();
-                resumeToken = resumeTokenDoc.getString("_data").getValue();
```

Comment:
Dummy string. What "_data" is?
It is also repeated many times

Reply:
Corrected in https://github.com/gravity9-tech/mongocdc/pull/36/files